dataline-tests/src/acceptanceTests/java/io/dataline/test/acceptance/AcceptanceTests.java

```diff
@@ -115,21 +116,42 @@ public void fullTestRun() throws IOException, ApiException, SQLException {
     // select all columns
     schema.getTables().forEach(table -> table.getColumns().forEach(c -> c.setSelected(true)));
 
-    ConnectionRead createdConnection = testCreateConnection(createdSourceImplId, createdDestinationImplId, schema);
+    ConnectionRead createdConnection = testCreateConnection(createdSourceImplId, createdDestinationImplId, schema, 1L);
 
     testRunManualSync(createdConnection.getConnectionId());
 
+    // TODO This is a bit of a hack to get around the fact that we don't have incremental behavior.
```

Comment:
This hack wouldn't be necessary if your tests were running with some sort of isolation.

Two options:
- Either for every tests you can generate a random database name and configure your source/target accordingly (might not be possible since today we only support one destination)
- Or you stop and restart your source/target postgres container for every test.

During the teardown of your tests you should make sure you "disable" the connection you created for the test you just run